zio/csvio/reader_test.go

```diff
@@ -10,7 +10,7 @@ import (
 
 func TestNewReaderUsesContextParameter(t *testing.T) {
 	zctx := zed.NewContext()
-	rec, err := NewReader(zctx, strings.NewReader("f\n1\n")).Read()
+	rec, err := NewReader(zctx, strings.NewReader("f\n1\n"), ReaderOpts{','}).Read()
```

Comment:
```suggestion
	rec, err := NewReader(zctx, strings.NewReader("f\n1\n"), ReaderOpts{}).Read()
```

Reply:
`ReaderOpts{}` was causing errors `(csv: invalid field or comment delimiter)` when running`make test-unit`:

Reply:
Yes, I should have mentioned that making the zero-value change in NewReader will get this passing with just ReaderOpts{}.